src/views/wiki/edition/utils/document-edition-view-mixin.js

```diff
@@ -168,19 +168,6 @@ export default {
       }
     },
 
-    setGeometryPoint() {
```

Comment:
Removing this function break the edition using numbers inputs now ?

Reply:
What do you mean exactly ? Not sure to understand, got no error and everything seems working fine

Reply:
@cbeauchesne ?